io.github.bothlab.syntalos.yaml

```diff
@@ -0,0 +1,262 @@
+id: io.github.bothlab.syntalos
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: syntalos
+rename-icon: syntalos
+rename-desktop-file: syntalos.desktop
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=all
+  - --filesystem=host
+  - --socket=system-bus
+  - --socket=session-bus
```

Comment:
allowing specific names a much prefered

Reply:
The user may want to write code which makes DBus calls to other applications on the system (like custom-built external applications) that we have no control over. So we don't know in advance which DBus interfaces are needed.
To filter interfaces, Syntalos also would need to be changed to e.g. not speak to RtKit directly but through a portal, which is also quite some work that needs to be done.

Reply:
you can explicitely allow RTKit. Also RTKit doesn't work through the sandbox you have to use the portal. Same DBus function, but on a different interface, and on the session bus.

Reply:
Exactly, it requires changes on existing code to work in the sandbox. RtKit is also not the only interface we use, and I have no idea what code other researchers are running as modules of the application - if it does DBus calls (which is actually very likely), limiting DBus access will break their custom code and result in a harder time to migrate to the Flatpak'ed Syntalos build, which is not what I want.
This application can run full, complex Python code to talk to other programs on the same machine, do online analysis of incoming data, control external hardware devices, etc.
I would very much prefer to at least initially have very few restrictions of the sandbox and then later see if we can tighten them at all (we'd still need to be able to run `flatpak-spawn` though for the "runcmd" module, and at that point limiting DBus access feels quite futile).